.github/CONTRIBUTING.md

```
@@ -64,45 +64,32 @@ bug if it ever happens again. This prevents regressions from sneaking in.
 
 - Run `npm start` in one terminal. This watches the source tree and compiles it incrementally.
-- When desired, run `npm run test` in another terminal. This runs tests using the compilation output from `npm start`.
-- Run `npm run test:debug` to use headful DevTools to debug or develop tests
+- Run `npm run test unit` to run unit tests, or `npm run test visual` to run visual regression tests.
+- Run `npm run test unit debug` to use headful DevTools to debug or develop tests
 
 #### Visual Regression Testing
 
 PixiJS uses a custom visual tester that allows you to create pixi scenes and compare them to a reference image.
-These tests can be found [here](../tests/visual/scenes/). To run these tests, run `npm run test:scene` from the command line, or run `npm run test:scene:debug` to use headful DevTools to debug or develop tests.
+These tests can be found [here](../tests/visual/scenes/). To run these tests, run `npm run test visual` from the command line, or run `npm run test visual debug` to use headful DevTools to debug or develop tests.
```

markdownlint-cli2 (0.20.0)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)